datamodel_code_generator/model/pydantic/imports.py

```diff
@@ -17,8 +17,8 @@
 IMPORT_UUID4 = Import.from_full_path('pydantic.UUID4')
 IMPORT_UUID5 = Import.from_full_path('pydantic.UUID5')
 IMPORT_ANYURL = Import.from_full_path('pydantic.AnyUrl')
-IMPORT_IPV4ADDRESS = Import.from_full_path('pydantic.IPv4Address')
-IMPORT_IPV6ADDRESS = Import.from_full_path('pydantic.IPv6Address')
+IMPORT_IPV4ADDRESS = Import.from_full_path('pydantic.validators.IPv4Address')
+IMPORT_IPV6ADDRESS = Import.from_full_path('pydantic.validators.IPv6Address')
```

Comment:
@goodoldneon 
In official documents ,It's written `ipaddress.IPv4Address` ,`ipaddress.IPv6Address`
https://pydantic-docs.helpmanual.io/usage/types/#pydantic-types
I have not used it.
Where should we import it from? 🤔

Reply:
Oh, it looks like it should be imported from the `ipaddress` lib. It isn't defined by Pydantic:

https://github.com/samuelcolvin/pydantic/blob/5261fd05a0374b84ce2602d45990baf480fa2417/pydantic/validators.py

That's probably why the docs say `ipaddress.IPv4Address`

Reply:
OK, Would you please change it?